<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAddPrefixSuffixWidget</class>
 <widget class="QWidget" name="CAddPrefixSuffixWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Prefix and/or Suffix into Each Line</string>
  </property>
  <widget class="QTextEdit" name="m_oTextEditInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>780</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_oLabelPrefix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>521</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Add this prefix into the beginning of each line:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_oLineEditPrefix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>780</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_oLabelSuffix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>521</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Add this prefix into the beginning of each line:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_oLineEditSuffix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>780</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_oTextEditOutput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>780</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_oBtnLoadFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>LOAD FILE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_oBtnBrowse">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_oLabelSelectedFile">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>470</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_oBtnAsveAs">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SAVE AS</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_oRadioBtnUnix">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>470</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Unix</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_oRadioBtnDos">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>470</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Dos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_oBtnAddPrefixSuffix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ADD PREFIX AND/OR SUFFIX</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_oBtnSellectAll">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>33</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_oBtnClearAll">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>33</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_oLineEditSaveAs">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>470</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
